### PR TITLE
Plantain Solr Configuration

### PR DESCRIPTION
**Pull request by tpendragon**

### CODE REVIEW

File: roles/pulibrary.solr/tasks/main.yml [hunk 1]
```diff
@@ -26,12 +26,13 @@
   when: solr_install_path_status.stat.isdir is not defined
   register: solr_download_status
 
-- name: Expand Solr.  # noqa 503
+- name: Expand Solr.  # noqa 503 ANSIBLE0016
```
Comment (hackmastera) [inline roles/pulibrary.solr/tasks/main.yml]: what is this?

File: playbooks/plantain_staging.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@
     - role: roles/pulibrary.nodejs
     - role: roles/pulibrary.openjdk
     - role: roles/pulibrary.blacklight-app
```
Comment (jrgriffiniii) [inline playbooks/plantain_staging.yml]: Might this be added as a dependency to the `pulibrary.plantain` role? I would contend that its own dependencies should be run for each `plantain` server provisioning.

File: group_vars/plantain/plantain_staging.yml [hunk 3]
```diff
@@ -45,6 +50,6 @@ rails_app_vars:
   - name: APPLICATION_HOST_PROTOCOL
     value: '{{application_host_protocol}}'
   - name: SOLR_URL
-    value: 'http://lib-solr-staging.princeton.edu:8983/solr/pulfa3-staging'
+    value: "http://localhost:8983/solr/{{ plantain_solr_core }}"
```
Comment (jrgriffiniii) [inline group_vars/plantain/plantain_staging.yml]: These aren't being set for the user `deploy`, I am working to diagnose and fix these now.

File: roles/pulibrary.solr/tasks/main.yml (hunk 1 above)
Comment (kayiwa) [reply roles/pulibrary.solr/tasks/main.yml]: I added the `.ansible-lint` which is the pattern used in other roles @hackmastera

**Pull request merged**